### PR TITLE
[Fix-697] adding missing key to elements array in AccidentsOccurred

### LINKED ISSUE #697: [Bug] key is missing in AccidentsOccurred
**Describe the bug**

**To Reproduce**
2. see error in console:
```
index.js:1 Warning: Each child in a list should have a unique "key" prop.

Check the render method of `AccidentsOccurred`. See https://fb.me/react-warning-keys for more information.
    in Title1 (at TextViewHeader.tsx:35)
    in AccidentsOccurred (at TextViewHeader.tsx:80)
    in div (created by Styled(MuiBox))
...
```

**Screenshots**
image

**Environment**
Browser: Chrome
Environment: Local

**Additional context**
This is happening because some elements in `elements` array missing `key`

### PR DESCRIPTION
**Pull request by IsaacM890**
Hi, 
its my first issue on this project , and i'm still learning the code , i just want to know if im on the right direction with this issue , thanks 

close #697

### CODE REVIEW

File: src/components/molecules/TextView/TextViewHeader.tsx [hunk 1]
```diff
@@ -32,8 +32,10 @@ const AccidentsOccurred: FC<AProps> = ({ accidentsCount, singleType }) => {
     <Box mr={1} key={1}>
       <Typography.Body1>{t('textView.occurred')}</Typography.Body1>
     </Box>,
-    <Typography.Title1 bold>
-      <Box className={classes.numOfAcc}>{accidentsCount}</Box>
+    <Typography.Title1 key={2} bold>
+      <Box mr={1} className={classes.numOfAcc}>
```
Comment (yuvalbl) [inline src/components/molecules/TextView/TextViewHeader.tsx:36]: remove ` mr={1}`

File: package.json [hunk 2]
```diff
@@ -77,11 +77,11 @@
   },
   "devDependencies": {
     "@storybook/addon-actions": "^6.1.16",
-    "@storybook/addon-essentials": "^6.1.16",
+    "@storybook/addon-essentials": "^6.3.4",
```
Comment (yuvalbl) [inline package.json:80]: these changes should not be part of this PR

**Review by yuvalbl — APPROVED** [review]

File: package.json (hunk 2 above)
Comment (IsaacM890) [reply package.json]: I changed them back on the last commit

File: package.json [hunk 3]
```diff
@@ -77,11 +77,11 @@
   },
   "devDependencies": {
     "@storybook/addon-actions": "^6.1.16",
-    "@storybook/addon-essentials": "^6.3.4",
+    "@storybook/addon-essentials": "^6.1.16",
     "@storybook/addon-links": "^6.1.16",
     "@storybook/node-logger": "^6.1.16",
     "@storybook/preset-create-react-app": "^3.1.5",
-    "@storybook/react": "^6.3.4",
+    "@storybook/react": "^6.1.16",
```
Comment (almogbhl) [inline package.json]: what's the reason for those changes?

**Review by yuvalbl — APPROVED** [review]

**Comment by yuvalbl** [comment]
replaced by #723
